modules/geopackage/src/geopackage-loader.ts

```diff
@@ -30,6 +31,9 @@ export const GeoPackageLoader: LoaderWithParser = {
   options: {
     geopackage: {
       sqlJsCDN: 'https://sql.js.org/dist/'
+    },
+    gis: {
```

Comment:
I wonder if we could deprecate this 'gis' option and replace it with `shape`?

Reply:
Either way. I guess `shape` is more general for non-gis loaders?

Reply:
This is probably best left for a future PR where we can change all loaders to use `shape` instead of `options.gis.format` at once?